develop/reference_rollups.rst

```diff
@@ -0,0 +1,83 @@
+.. _rollups:
+
+Caching Aggregations with Rollups
```

Comment:
This section doesn't quite belong in the 'reference' section for SQL. Is there anything here that isn't covered in our real-time use-case guide? Maybe ON_CONFLICT? 
I'm wondering if we nix this and just redirect as I mentioned above to the roll-up guide if we think this is covered elsewhere.

Reply:
Actually this section used to be buried in https://docs.citusdata.com/en/v7.4/develop/reference_dml.html#distributed-aggregations and had the title "Distributed Aggregations." It was deep inside "Inserting Data" so this moves it higher into its own section.

We could move it higher still, as a sibling of SQL Reference. I feel bad cutting out all that content.